Cargo.toml

```diff
@@ -12,10 +12,12 @@ members = [
 ]
 
 [workspace.dependencies]
-arrow = "54"
-alloy-dyn-abi = "0.8"
-alloy-json-abi = "0.8"
-alloy-primitives = "0.8"
+arrow = "54.2.1"
```

Comment:
Don't need anything after 54 because 54 means "^54.0.0" which means use the latest semver compatible version with "54.0.0" so it will use "54.2.1" if it is latest automatically.

When "54.2.1" is specified it might use any higher version that is "54.x.x" so specifiying more than 54 can be misleading

Reply:
Same idea with "0.8.x" (only need "0.8") and so on